drivers/hud_apr/app/models/hud_apr/generators/shared/fy2024/dq/question_three.rb

```diff
@@ -57,7 +57,7 @@ def q3_veteran_row(sheet, universe_members)
       )
 
       total_cell = sheet.update_cell_value(cell: 'E2', value: [dkpntr_cell, missing_cell, issue_cell].map(&:value).sum)
-      total_cell.add_members([dkpntr_cell, missing_cell, issue_cell].map(&:members).sum)
+      total_cell.add_members([dkpntr_cell, missing_cell, issue_cell].map(&:members).sum([]))
```

Comment:
using sum() on arrays with non-numeric values is deprecated unless an initial value is provided

Reply:
it looks like we are  getting arrays back here which requires a default arg